.github/workflows/pr.yml

```diff
@@ -11,56 +11,51 @@ on:
         description: 'Reason for manual run'
         required: false
 
+concurrency:
+  group: build-${{ github.ref }}
+  cancel-in-progress: true
+
 jobs:
   gradle:
     strategy:
       matrix:
-        os: [ubuntu-latest]
+        os: [ ubuntu-latest ]
```

Comment:
I think this can be simplified to run on `ubuntu-latest` directly as it's currently not really taking advantage of the matrix strategy, but I left it as is for now

Reply:
Every now and then someone complains about a Windows-specific failure and I'm _almost_ sympathetic enough to try to run CI on Windows, but I think that would be a big lift, absent funding that is 🙃